cookbooks/bcpc/attributes/graphite.rb

```diff
@@ -2,8 +2,10 @@
 default['bcpc']['graphite']['relay_port'] = 2013
 default['bcpc']['graphite']['web_port'] = 8888
 default['bcpc']['graphite']['log']['retention'] = 15
+default['bcpc']['graphite']['data']['retention'] = 15
 default['bcpc']['graphite']['timezone'] = "'America/New_York'"
 default['bcpc']['graphite']['local_data_dir'] = "/opt/graphite/storage/whisper"
+default['bcpc']['graphite']['local_log_dir'] = "/opt/graphite/storage/log"
```

Comment:
I don't understand what you mean.  Graphite and carbon already write logs to the correct location